src/main/webapp/maps.html

```diff
@@ -18,12 +18,12 @@
       padding: 10px;
     }
   </style>
-  <script src="http://code.jquery.com/jquery-1.11.3.min.js"></script>
   <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBxd60VBUhGDGcDgEEwgyFkEfVtcG6xiZU"></script>
+  <script src="/js/markerclusterer.js"></script>
```

Comment:
Hi Reine, the `markerclusterer.js` file is not present in the repository yet. Maybe just remove it for this PR.

Reply:
Will do